style.css

```diff
@@ -88,19 +85,27 @@ li {
   font-style: normal;
   font-weight: 700;
   font-size: 40px;
-  line-height: 60px;
+  line-height: 52px;
   color: #172b4d;
   margin: 0 10px;
+
+  /* transition: transform  4s cubic-bezier(.57,.64,1,.01); */
 }
 
+/* .heading:hover {
+  color: indigo;
+  transform: scaleX(1.1) translate(70px);
+  font-style: italic;
+  font-size: 35px;
+} */
```

Comment:
- [ ] Please remove all commented code in your documents for a cleaner document.